COM-2003 Added French text fixed few text issues

hrishid opened the pull request:
COM-2003 Changed Template for gateway-Giftcard-created

COM-2003 email template for gatewayGiftCard

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -239,6 +239,10 @@
   "fullDesc": "Description",
+  "gatewayGiftCardDeleteEmail": "DO NOT DELETE THIS EMAIL",
+  "gatewayGiftCardEmailIntro": "Thank you for purchasing a {0} Gift Card. Do not delete this message -- to redeem the gift card you must use the gift card claim code below. You may want to print a copy of this message for easy reference later.",
+  "gatewayGiftCardNeedsCard": " You will need the card to complete your purchase" ,
```
Text doesn't match what's in the jira issue. Should say `card number`.

hrishid replied:
Changed the text and squashed commits that's why its saying outdated.

michaelsadlonkibo commented on labels/de-DE.json:
```diff
@@ -181,6 +181,10 @@
   "fullDesc": "Beschreibung",
+  "gatewayGiftCardDeleteEmail":"NE SUPPRIMEZ PAS CET EMAIL",
+  "gatewayGiftCardEmailIntro":"Merci d'avoir acheté une {0} carte-cadeau. Ne supprimez pas ce message - pour utiliser la carte-cadeau, vous devez utiliser le code de réclamation de la carte-cadeau ci-dessous. Vous souhaiterez peut-être imprimer une copie de ce message pour une référence ultérieure.",
+  "gatewayGiftCardNeedsCard": " Vous aurez besoin de la carte pour finaliser votre achat",
```
Verify text is correct per english.

Commit:
COM-2003 FIxed review Comments changed text and $ sign at GiftCardEmailAmmount

COM-2003 Added French text fixed few text issues

COM-2003 Changed Template for gateway-Giftcard-created

COM-2003 email template for gatewayGiftCard

COM-2003 Fixed Currency
